theme/js/components/organization/request-membership.vue

```diff
@@ -0,0 +1,48 @@
+<!--
+---
+name: RequestMembership
+category: Interactions
+---
+
+# Request Membership
+
+A simple request membership prompt.
+-->
+
+<template>
+    <li class="my-md">
+        <a @click.prevent="JoinOrga"
+      class="nav-link">{{ $t('Ask to join the organization as a producer') }}</a>
```

Comment:
Was that the previous wording?

Reply:
Nope, it used to be "join/rejoindre" only in legacy. I used the suggestion in https://github.com/etalab/data.gouv.fr/issues/438#issuecomment-915266108.